inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -859,6 +885,22 @@ private boolean hasImplicitNamespace(String s)
         }
         return false;
     }
+    
+    public boolean isIdentiferBaseProperty(KBHandle handle)
+    {
+
+        List<IRI> listIRI = new ArrayList<IRI>();
```

Comment:
`new ArrayList<IRI>()`-> `new ArrayList<>()` - btw. you do this a lot, the generic type is not required in these cases - Java can automatically infer it.

Reply:
Yeah, its an old school habit ;) Will remove it.